Readme.md

```diff
@@ -8,7 +10,91 @@ Do you want to quickly try out this project and see it in action? Check it out i
 
 ## Documentation
 
-Documentation and samples coming soon!
+### Async Editor
+
+The Async Editor feature provides an infrastructure for creating editors that can load and save content asynchronously. The editor can optionally render multiple pages where each page can load and save data independently.
+
+#### Creating an Async Editor for content items
```

Comment:
Why? As a developer, I'd appreciate seeing some guide in the main readme rather than figuring out there's a Sample project.

Reply:
You shouldn't have to figure out the sample project, or if there is one, since it should be apparent from this Readme.

The sample project, with its walkthrough, serves the purpose of instructing the developer how to utilize the module. That should be written in a way that it completely conveys what's written in this block, in a way that's at least as easy to digest. However, using living code samples has the benefit that it's actually code, and thus not suspect to breaking during refactoring or any other API changes, as opposed to code in an MD file that we have to take care of keeping up-to-date manually, and there's no automated testing for it.